.github/workflows/release.yml

```
@@ -0,0 +1,72 @@
+name: relase-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+        npmTag:
+          description: 'NPM tag to release'
+          required: true
+          default: 'next'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+    env:
+      CI: true
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v2-beta
+        with:
+          always-auth: true
+          node-version: 12
```

What about using the latest LTS here? Can't we specify the version as `latest` or sth?

```suggestion
          node-version: 14
```

---

Actually not, their docs says:

```
It will first check the local cache for a semver match. The hosted images have been updated with the latest of each LTS from v8, v10, v12, and v14.
```